### ISSUE TITLE
Should be possible to host on subdomain in SINGLE_TENANT_MODE

### ISSUE THREAD

**Issue by Sinetheta** [DefactoSoftware/Hours#262]
Lines like `root_url(subdomain: current_subdomain)` in conjunction with the [email_helper](https://github.com/DefactoSoftware/Hours/blob/master/app/helpers/email_helper.rb) mean that if you are hosting in SINGLE_TENANT_MODE then your subdomain will be trimmed off.

**Comment by jurre** [comment]
Good point, thanks! Feel free to open a PR with a fix, otherwise I'll look into it asap.
